osu.Game.Benchmarks/BenchmarkRealmReads.cs

```diff
@@ -27,9 +27,9 @@ public override void SetUp()
             storage = new TemporaryNativeStorage("realm-benchmark");
             storage.DeleteDirectory(string.Empty);
 
-            realmFactory = new RealmContextFactory(storage, "client");
+            realm = new RealmAccess(storage, "client");
 
-            realmFactory.Run(realm =>
+            realm.Run(realm =>
```

Comment:
Some places in this pull look to have been changed to `realm.Run(r => ...)`, but this and a large number of other places (tens to hundreds left in this pull alone) continue (ab)using shadowing by writing `realm.Run(realm => ...)`. I presume this is just an oversight and another pass to fully eliminate the `realm.Run(realm => ...)` pattern is in order? Not a fan of name shadowing ever in general.

Reply:
interesting that these don't trigger inspections..

Reply:
I've updated all remaining cases I could find. Still not 100% on the `r` variable, sometimes it feels a bit hard to follow, but let's go with it for now.